types/openpay/index.d.ts

```diff
@@ -0,0 +1,282 @@
+// Type definitions for openpay 1.0
+// Project: https://github.com/open-pay/openpay-node
+// Definitions by: Bruno Jardón <https://github.com/bjardon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace openpay;
```

Comment:
When using `export as namespace`, it's nice to add a corresponding test consuming the types that way.

Reply:
The module I'm writing types for is not a UMD module. I believe I left export by mistake.